config.go

```diff
@@ -148,3 +148,20 @@ func dataDir() (string, error) {
 	}
 	return filepath.Join(home, ".local", "share", opBotConfigDir), nil
 }
+
+func createDataDirIfDoesntExist() (string, error) {
```

Comment:
We don't need this function. os.MkdirAll will already create a directory and return nil if it already exists. So, just replace the call to this function by os.MkdirAll(...) and that's it.